README.md

```diff
@@ -1,31 +1,42 @@
-# curriculum-databases-projects-template
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# Leaderboard
```

Comment:
- [ ] Kindly add a detailed description of your project where users get to know about your project features and more.

Reply:
Done!